### ISSUE TITLE
Questions about data generation

### ISSUE THREAD

**Issue by qsisi** [fabiopoiesi/dip#17]
Thanks for your sharing, here I have a question about how to generate 3DMatch_train.zip data.

As far as I know, the data downloaded from 3DMatch contains 62 RGB-D sequences, do those *.ply files were generated by TSDF fusion provided by 3DMatch-toolbox?

Thanks for your help.

**Comment by fabiopoiesi** [comment]
Correct, [this](https://github.com/andyzeng/3dmatch-toolbox#training-3dmatch-from-rgb-d-reconstructions) is the tool.
In my case I used the ones provided by this [paper](https://github.com/zgojcic/3DSmoothNet), I didn't recompute them.

**Comment by qsisi** [comment]
Thanks for your reply, I could not correctly use the tool provided by 3DMatch due to the opencv configuration :( 

It seems that this info: "**3DMatch_train/7-scenes-fire/cloud_bin_19.info.txt**" is missing in **3DMatch_train.zip** ?

**Comment by qsisi** [comment]
I got a new question :)

The **3DMatch_train.zip** contains 82 folders, which in my understanding, includes the whole training scenes in 3DMatch, according to the [split.txt](https://vision.princeton.edu/projects/2016/3DMatch/downloads/rgbd-datasets/split.txt) provided in the original 3DMatch website, there are total 54 scenes for training.

So it means, 54 scenes of rgbd-sequences are converted into 82 folders of *.ply files? (**54 scenes -> 82 folders**)

But when I check the [fuseSceneFragments.m](https://github.com/andyzeng/3dmatch-toolbox/blob/master/depth-fusion/fuseSceneFragments.m), it seems that it just fuse each scene into one folder, so maybe **54 scenes -> 54 folders**?

So why the **3DMatch_train.zip** contains **82** folders? Maybe they are not generated using the [3DMatch-toolbox](https://github.com/andyzeng/3dmatch-toolbox#training-3dmatch-from-rgb-d-reconstructions)?

Thank you very much for your help!
